.gitignore

```diff
@@ -1,2 +1,3 @@
 /build
 /twet
+.idea
```

Comment:
Pls dont do this. I suggest to keep editor specific ignores and such in a global git excludesfiles in your setup.

Reply:
Sure :)